vips/image_test.go

```diff
@@ -97,8 +97,8 @@ func TestImageRef_Error(t *testing.T) {
 
 	src := bytes.NewReader(srcBytes)
 	img, err := NewImageFromReader(src)
-	assert.Error(t, err)
-	assert.Nil(t, img)
+	assert.NoError(t, err)
```

Comment:
why the drastic change in assertion?

Reply:
Because libvips now handles PNG with oversized meta gracefully instead of failing